pygeos/creation.py

```diff
@@ -64,7 +64,9 @@ def points(coords, y=None, z=None, indices=None, **kwargs):
     if indices is None:
         return lib.points(coords, **kwargs)
     else:
-        return simple_geometries_1d(coords, indices, GeometryType.POINT)
+        return simple_geometries_1d(
+            coords, indices, GeometryType.POINT, out=kwargs.get("out")
```

Comment:
Is there are a reason for not making `out` a top-level parameter for `points` (or other top-level functions below)?

Reply:
I added it and fixed the docstrings.